ci-operator/config/openshift/baremetal-runtimecfg/openshift-baremetal-runtimecfg-master.yaml

```diff
@@ -62,12 +62,6 @@ tests:
   commands: make test
   container:
     from: src
-- as: e2e-metal-ipi
```

Comment:
I don't think we want to remove these jobs completely. They provide our ipv4 coverage in addition to SDN. I believe we can just switch them to the baremetalds-e2e-ovn-ipv4 workflow.

Reply:
that sounds right. I've updated the PR for this. thanks for the comment.